resources/public/api/conf/1.0/other_expenses_delete.yaml

```diff
@@ -0,0 +1,83 @@
+summary: Delete Other Expenses
+description: |
+  This endpoint enables you to delete other expenses.
+  A National Insurance number and tax year must be provided.
```

Comment:
The description should be `This endpoint allows a developer to delete expenses income for trade union and patent royalties. A National Insurance number and tax year must be provided.` according to the spec

Reply:
That's from the old spec